requirements.txt

```diff
@@ -1,63 +1,61 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.12
 # by the following command:
 #
-#    pip-compile --annotation-style=line --output-file=requirements.txt requirements.in setup.py
+#    pip-compile --annotation-style=line --output-file=requirements.txt setup.py
```

Comment:
Be careful that the file needs to be regenerated taking into account `requirements.in`, otherwise the `pygraphviz` dependency will be missing:
```
pip-compile --annotation-style=line --output-file=requirements.txt requirements.in setup.py
```